### PR TITLE
CRUD for events using postgres

### PR DESCRIPTION
**Pull request by frankip**
#### What does this PR do?
Switch from Data structure to Postgressql
implement event CRUD functionalities with Postgres
#### Description of Task to be completed?
Create migrate file for sqlalchemy
create and get events
get single events update and delete event
#### How should this be manually tested?
clone the repo and change the branch to challenge-3
create a Postgres database and go to config file in 
the TestingConfig file change the SQLALCHEMY_DATABASE_URI 
to the one you created
then run 
```nosetests test```
#### What are the relevant pivotal tracker stories?

#153045245
#153045224

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -2,18 +2,25 @@
...
 
+# initialize sql-alchemy
+db = SQLAlchemy()
+
 # Initialize the app
 app = FlaskAPI(__name__, instance_relative_config=True)
-app.secret_key = "string"
+# Load the config file
+app.config.from_object(app_config['development'])
+app.secret_key = "secret_string"
```
Comment (Rodgers-M) [inline app/__init__.py]: consider getting  the secret key from an environment variable.

File: app/models.py [hunk 2]
```diff
@@ -55,28 +56,50 @@ def verify_password(self, password):
...
     """
-    events_db = {}
-    id_generator = itertools.count(1)
+    # events_db = {}
+    # id_generator = itertools.count(1)
+
+    __tablename__ = 'events_db'
+
+    id = db.Column(db.Integer, primary_key=True)
+    event = db.Column(db.String(255))
```
Comment (Rodgers-M) [inline app/models.py]: What attribute of this class is stored in this field?

**Comment by meshnesh** [comment]
Good implementation of the tests
Shouldn't be having Tokens when logging in a person?
Good implementation of the tables

File: app/__init__.py (hunk 1 above)
Comment (frankip) [reply app/__init__.py]: sure

**Pull request merged**